docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -290,6 +265,15 @@ Release
 
    Wait for the build to complete successfully.
 
+#. Set up ``AWS Access Key ID`` and ``AWS Secret Access Key`` Amazon S3 credentials:
```

Comment:
If it is copied automatically from the host system, then setting it up (on the host) is a step that can be done one time per host-machine, so it doesn't need to be a step in the release process.